src/core/Present.jl

```diff
@@ -117,6 +117,10 @@ function add_definition!(pres::Presentation, name::Symbol, rhs::GATExpr)
   generator
 end
 
+""" Get the index of a generator
+"""
+index_of(pres::Presentation, x::Symbol) = pres.generator_name_index[x].second
```

Comment:
Let's call this `generator_index` or `index_of_generator` since `index_of` sounds very generic.